src/lib/connectors/returns.js

```diff
@@ -124,6 +124,19 @@ const getLinesForReturn = async ret => {
   return linesClient.findAll(linesFilter);
 };
 
+/**
+ * Gets the returns KPI data by cycle within the specified parameters
+ * @param {Date} startDate
+ * @param {Date} endDate
+ * @param {Boolean} isSummer
+ * @returns Object
+ */
+const getKPIReturnsByCycle = async (startDate, endDate, isSummer) => {
```

Comment:
Not sure async is needed here.

Plus could also be `const qs = querystring.stringify({startDate, endDate, isSummer })`

Reply:
changed thanks